docs/Tags.md

```diff
@@ -1,4 +1,48 @@
+[DE](DE%3ATags.md)
+
 # Tags and Tag-groups
 
-***
-Switch Language: [DE](DE%3ATags.md)
\ No newline at end of file
+## Creating time bookings with tags
+
+Time bookings can be created with one or multiple `tags`. Tags are intended to help identify a time booking and also associate multiple time bookings with each other.
```

Comment:
Isn't none possible?

Reply:
yes, you can of course create time bookings without any tags